database/dict_converters/media_converter.py

```diff
@@ -20,9 +20,18 @@ def mediasConverter_v3(cls, medias):
 
     @classmethod
     def mediaConverter_v3(cls, media):
-        return {
+        dict = {
             'type': media.slug_name,
             'foreign_key': media.foreign_key,
             'details': media.details if media.details else {},
             'preferred': True if media.preferred_references != [] else False,
+            'view_url': None,
+            'direct_url': None,
         }
+        if media.slug_name == "youtube":
+            dict["direct_url"] = media.youtube_url
```

Comment:
For images, we actually want this to be the thumbnail url, see https://github.com/the-blue-alliance/the-blue-alliance-android/blob/5c94e11e5640a08a3547c16646efec3f018835c8/android/src/main/java/com/thebluealliance/androidclient/types/MediaType.java#L57

So `"http://img.youtube.com/vi/{foreignKey}/hqdefault.jpg"`

Reply:
Resolved in the latest commit.